app/views/content/non-uk-teachers/ukraine.md

```diff
@@ -87,26 +100,66 @@ We understand it might not be possible to provide all this evidence. However, it
+
+ 
+You can also learn English with a private provider. Well-respected English language courses include: 
+
+* [International English Language Testing System (IELTS)](https://www.ielts.org/)
+* [Test of English as a Foreign Language (TOEFL)](https://www.ets.org/toefl)
+* [C2 Proficiency, formerly known as Cambridge English: Proficiency (CPE)](https://www.cambridgeenglish.org/exams-and-tests/proficiency/)
+
+
 
 [Find out more about teaching in England if you trained overseas](/non-uk-teachers/teach-in-england-if-you-trained-overseas). 
 
 ### Apply for a teaching job
 
-To find a teaching job, you must apply to individual schools (not DfE). Use the [Teaching Vacancies](https://teaching-vacancies.service.gov.uk/) service to search for jobs by region, subject specialism and age group. This service also lists [teaching assistant](https://nationalcareers.service.gov.uk/job-profiles/teaching-assistant) and education support roles. It can be helpful to gain experience of the English education system working in a teaching assistant or support role before you apply for teaching jobs. 
+To find a teaching job, you must apply to individual schools (not DfE). Use the [Teaching Vacancies](https://teaching-vacancies.service.gov.uk/) service to search for jobs by region, subject specialism and age group. 
+
+This service also lists [teaching assistant](https://nationalcareers.service.gov.uk/job-profiles/teaching-assistant) and education support roles. It can be helpful to gain experience of the English education system working in a teaching assistant or support role before you apply for teaching jobs. 
 
 For a short-term unpaid placement, you can [request school experience in a primary or secondary school in England](https://schoolexperience.education.gov.uk/). 
 
 Most teaching vacancies are advertised in March, April or the first half of May, with jobs starting at the beginning of the school year in September. Set up a [Teaching Vacancies](https://teaching-vacancies.service.gov.uk/) job alert to be updated about opportunities throughout the year.
 
+### Other teaching job options
+
+Search online for 'supply teacher' to register with an agency for short-term roles in school. Supply teachers fill in when schools are temporarily short of staff.  
```

Comment:
Is this a clear enough instruction for users? We can't guarantee which search results they'll see if they type 'supply teacher' so won't know if they're going to the right place. Is there not something more specific we can link them to?

Reply:
@laurahermionetennant92 have you had any more thoughts about this?